woocommerce/woocommerce
# Revert update to createRoot in Checkout block.

This PR reverts #48843 because the new update caused some setState race conditions in some payment plugins, we will bring this back in once we fully identify the issue.

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:








### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Install and setup Square, [and create a sandbox, follow this document.](https://woocommerce.com/document/woocommerce-square/testing-the-woocommerce-square-extension-in-sandbox-mode/)
2. Once fully setup, go to Checkout, select Cash app.
3. Hit the cashapp button, scan the QR code with your phone, a window will open on your phone, click approve.
4. Make sure the order passes.



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 
Revert update to React 18 in Checkout block.

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
